src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,181 @@
 package core.basesyntax;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final short GROWTH_FACTOR = 2;
+    private int threshold = 12; //initial threshold value
+    private int size;
+    private int capacity;
```

Comment:
this value is the same as table.length

Reply:
I removed this value and used table.length everywhere